__docs__/wonder-blocks-link/link.stories.tsx

```diff
@@ -609,18 +597,19 @@ InlineLight.play = async ({canvasElement}) => {
         `text-decoration: underline ${color.white} solid`,
```

Comment:
**suggestion:** I looks like this is also flaky. I'd suggest commenting the whole `play` function.